functions/package.json

```diff
@@ -14,6 +14,24 @@
   },
   "main": "lib/index.js",
   "dependencies": {
+    "@angular/animations": "~9.0.2",
```

Comment:
Because of curiosity: Why do we need to add angular dependencies, do we have any articles mentioning that?